test/codes_tests/test_c_implementation_simplified.py

```diff
@@ -3,6 +3,7 @@
 from amuse.support.interface import InCodeComponentImplementation
 
 from amuse.test.amusetest import TestWithMPI
+import amuse.test.compile_tools as compile_tools
```

Comment:
minor: import style

Reply:
What would you prefer?

Reply:
there is no difference between them? I think they look best when all the same;-)